api-v2/src/plan/plan.service.ts

```diff
@@ -21,7 +20,7 @@ export class PlanService {
     try {
       return this.planRepository.save(newPlan);
     } catch (_error) {
-      throw new Error('Something went wrong when creating the new Plan');
+      return null;
```

Comment:
like you mentioned when we talked in person, here we lose information, and so it would be good to log this (sentry or something) later on. can you add a todo?